api/controllers/system.go

```diff
@@ -43,61 +23,37 @@ func SystemUpdate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 		return httperr.Server(err)
 	}
 
-	notifyData := map[string]string{}
-
-	if count := GetForm(r, "count"); count != "" {
-
-		count, err := strconv.Atoi(count)
+	// update based on form input
+	if cc := GetForm(r, "count"); cc != "" {
+		c, err := strconv.Atoi(cc)
 		if err != nil {
-			return httperr.Errorf(403, "count must be an integer")
+			return httperr.Errorf(403, "count must be numeric")
 		}
 
-		// -1 is an invalid value that indicates no change
-		if count != -1 {
-			if count <= 1 {
-				return httperr.Errorf(403, "count must be greater than 1")
-			}
-
-			stack, err := models.Provider().AppGet(rack.Name)
-			if err != nil {
-				return httperr.Errorf(404, "no such stack")
-			}
-
-			if stack.Parameters["Autoscale"] == "Yes" {
```

Comment:
Is this being checked elsewhere? I remember there being issues trying to set count with autoscale enabled.